README.md

```diff
@@ -11,7 +11,7 @@ Requirements
 ------------
 * Red Hat Enterprise / CentOS 5.5+ / Windows Server 2008+ (forwarder only) or Ubuntu LTS 12.04+
 * Chef 12+
-
+*
```

Comment:
This was made to kick off the travis build. 6dc58eb18050883af3d211291e2e554d62ed3fae
Let us revert it.